addons/mass_mailing/models/mailing.py

```diff
@@ -544,12 +545,19 @@ def _action_view_traces_filtered(self, view_filter):
 
     def action_view_clicked(self):
         model_name = self.env['ir.model']._get('link.tracker').display_name
+        recipient = self.env['ir.model']._get(self.mailing_model_real).display_name
+        helper_header = _("No %s clicked your mailing yet!", recipient)
+        helper_message = _("Link Trackers will measure how many times each link is clicked as well as "
+                           "the proportion of %s who clicked at least once in your mailing.", recipient)
         return {
             'name': model_name,
             'type': 'ir.actions.act_window',
             'view_mode': 'tree',
             'res_model': 'link.tracker',
             'domain': [('mass_mailing_id.id', '=', self.id)],
+            'help': Markup('<p class="o_view_nocontent_smiling_face">%s</p><p>%s</p>' % (
```

Comment:
Actually the variables should be outside of the Markup call, like ``Markup("<strong>%s</strong>") % "hello"`` (check other Markup use, probably to update accordingly :) )